src/Console/Commands/ForgeDeployCommand.php

```diff
@@ -69,7 +69,7 @@ public function handle()
         ])->post($deploymentUrl);
 
         if ($response->failed()) {
-            $this->error("Error using forge deployment trigger url. {$response->getStatusCode()}");
+            $this->error("Error using forge deployment trigger url. {$response->status()}");
```

Comment:
`getStatusCode()` does not seem to be a method on `\Illuminate\Http\Response`

Reply:
Good point, this is the guzzle client remains.